src/archiefvernietigingscomponent/templates/master.html

```diff
@@ -11,12 +11,17 @@
         <link href="https://fonts.googleapis.com/css?family=Fira+Sans:300,400,600|Material+Icons|Material+Icons+Outlined" rel="stylesheet">
 
         <link href="{% static 'css/screen.css' %}" media="screen" rel="stylesheet" type="text/css" />
+        <link href="{% static 'css/start_page/demo_tutorial.css' %}" media="screen" rel="stylesheet" type="text/css" />
```

Comment:
Sorry I missed it last time, but why is this css component not included into `screen.css' like all other custom css files?

Reply:
I imported the demo_tutorial.css into the screen.css!